audit-service/src/unit-tests/java/uk/gov/gchq/palisade/service/audit/service/LoggerAuditServiceTest.java

```diff
@@ -44,98 +52,88 @@
 
         // Then
-        assertThat(infoCaptor.getAllValues())
-                .isEmpty();
-        assertThat(errorCaptor.getAllValues())
-                .hasSize(2).contains(BAD_AUDIT_SUCCESS_MESSAGE, "user-service");
+        assertThat(infoCaptor).isEmpty();
+        assertThat(errorCaptor).containsOnly(BAD_AUDIT_SUCCESS_MESSAGE, USER_SERVICE.value);
     }
 
     @Test
     void testUserServiceAuditErrorMessage() {
+
         // Given
-        Mockito.doNothing().when(logger).error(Mockito.anyString(), errorCaptor.capture());
+        doNothing().when(logger).error(anyString(), errorCaptor.capture());
 
         // When
-        auditService.audit(ApplicationTestData.TEST_TOKEN, ApplicationTestData.auditErrorMessage(ServiceName.USER_SERVICE.value));
+        auditService.audit(TEST_TOKEN, auditErrorMessage(USER_SERVICE));
 
         // Then
-        assertThat(infoCaptor.getAllValues())
-                .isEmpty();
-        assertThat(errorCaptor.getAllValues())
-                .hasSize(1).first().isEqualTo(ApplicationTestData.auditErrorMessage("user-service"));
+        assertThat(infoCaptor).isEmpty();
+        assertThat(errorCaptor).containsOnly(auditErrorMessage(USER_SERVICE));
     }
 
     @Test
     void testResourceServiceAuditErrorMessage() {
+
         // Given
-        Mockito.doNothing().when(logger).error(Mockito.anyString(), errorCaptor.capture());
+        doNothing().when(logger).error(anyString(), errorCaptor.capture());
 
         // When
-        auditService.audit(ApplicationTestData.TEST_TOKEN, ApplicationTestData.auditErrorMessage(ServiceName.RESOURCE_SERVICE.value));
+        auditService.audit(TEST_TOKEN, auditErrorMessage(RESOURCE_SERVICE));
 
         // Then
-        assertThat(infoCaptor.getAllValues())
-                .isEmpty();
-        assertThat(errorCaptor.getAllValues())
-                .hasSize(1).first().isEqualTo(ApplicationTestData.auditErrorMessage("resource-service"));
+        assertThat(infoCaptor).isEmpty();
+        assertThat(errorCaptor).containsOnly(auditErrorMessage(RESOURCE_SERVICE));
     }
 
     @Test
     void testOtherServiceAuditErrorMessage() {
         // Given
-        Mockito.doNothing().when(logger).error(Mockito.anyString(), errorCaptor.capture());
+        doNothing().when(logger).error(anyString(), errorCaptor.capture());
 
         // When
-        auditService.audit(ApplicationTestData.TEST_TOKEN, ApplicationTestData.auditErrorMessage(ServiceName.POLICY_SERVICE.value));
+        auditService.audit(TEST_TOKEN, auditErrorMessage(POLICY_SERVICE));
 
         // Then
-        assertThat(infoCaptor.getAllValues())
-                .isEmpty();
-        assertThat(errorCaptor.getAllValues())
-                .hasSize(1).first().isEqualTo(ApplicationTestData.auditErrorMessage("policy-service"));
+        assertThat(infoCaptor).isEmpty();
+        assertThat(errorCaptor).containsOnly(auditErrorMessage(POLICY_SERVICE));
```

Comment:
id personally rather have assertions like
```
assertThat(infoCaptor)
                .as("")
                .extracting(ArgumentCaptor::getAllValues)
                .asList()
                .isEmpty();
        assertThat(errorCaptor)
                .as("")
                .extracting(ArgumentCaptor::getAllValues)
                .asList()
                .containsOnly(auditErrorMessage(POLICY_SERVICE));
```
Than have an assertions library that gives us no real benefit over assertJs native code, other than reducing duplication (which as a metric we arent tracking)

Also your assertions lack the .as() descriptions

Reply:
But isn't `assertThat(infoCaptor).containsOnly(xxxxxx));` easier to read?